testing/constraints-3.6.txt

```
@@ -14,3 +14,4 @@ packaging==14.3
 grpcio==1.33.2
 grpcio-gcp==0.2.2
 grpcio-gcp==0.2.2
+grpcio-status==1.33.2
```

```suggestion
grpcio-status==1.0.0
```

(to match the lower bound in setup.py)

---

Isn't the `grpcio-status` version supposed to be identical to the one for `grpcio`?